src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -256,6 +256,8 @@
     short-summary: The path to the template file.
   - name: --template-uri -u
     short-summary: The URI to the template file.
+  - name: --template-spec -t
```

Comment:
The existing short aliases for --template-file and --template-uri all use the first letter of the last words (-f, -u), so personally I prefer using -s over -t as the alias for --template-spec, because a) it's consistent with other parameters, b) it reminds me of "spec" (it's up to you to make the decision and I this shouldn't block the PR).

Reply:
That's a good idea, I will make the changes.